manifests/0000_10_kube-apiserver-operator_01_config.crd.yaml

```diff
@@ -12,3 +12,202 @@ spec:
     singular: kubeapiserver
   subresources:
     status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          properties:
+            failedRevisionLimit:
+              description: failedRevisionLimit is the number of failed static pod
+                installer revisions to keep on disk and in the api -1 = unlimited,
+                0 or unset = 5 (default)
+              format: int32
+              type: integer
+            forceRedeploymentReason:
+              description: forceRedeploymentReason can be used to force the redeployment
+                of the kube-apiserver by providing a unique string. This provides
+                a mechanism to kick a previously failed deployment and provide a reason
+                why you think it will work this time instead of failing again on the
+                same config.
+              type: string
+            logLevel:
+              description: logLevel is an intent based logging for an overall component.  It
+                does not give fine grained control, but it is a simple way to manage
+                coarse grained logging choices that operators have to interpret for
+                their operands.
+              type: string
+            managementState:
+              description: managementState indicates whether and how the operator
+                should manage the component
+              type: string
+            observedConfig:
+              description: observedConfig holds a sparse config that controller has
+                observed from the cluster state.  It exists in spec because it is
+                an input to the level for the operator +nullable
```

Comment:
note: this shouldn't be here, see https://github.com/openshift/kubernetes-sigs-controller-tools/pull/3#discussion_r261752842